luigi/scheduler.py

```diff
@@ -354,7 +356,7 @@ def __init__(self, worker_id, last_active=None):
         self.disabled = False
         self.rpc_messages = []
 
-    def add_info(self, info):
+    def update_info(self, info):
```

Comment:
I changed this method name, because I feel like it fits much better with what is actually happening.